fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb

```diff
@@ -90,7 +90,7 @@ def run!
           raise e
         else
           action_completed(@program[:name], status: FastlaneCore::ActionCompletionStatus::INTERRUPTED, exception: e)
-          puts("\nCancelled... use --verbose to show the stack trace")
+          abort("\nCancelled... use --verbose to show the stack trace")
```

Comment:
I think this is the right thing to do, will this break anybody's setups? I don't think so, worst case is it stops a lane from running due to exit code being 1, which is probably the right behavior.

Reply:
@taquitos This _shouldn't_ break anyone setup 😊 This only get's called if a user presses CONTROL+C so the user knows things are going to stop. This fixes an issue when a CI/worker/queue kills a job running fastlane where it actually returns an error like it should. This _may_ have some benefit forward fastlane.ci? Unsure since I don't know how those jobs are started up yet and what happens if manually cancelled 🙃